packages/appstore/src/constants/utils.ts

```diff
@@ -20,3 +21,96 @@ export const getHasDivider = (current_item_index: number, list_size: number, ava
             (list_size % available_grid_columns === 0 ? available_grid_columns : list_size % available_grid_columns)
     );
 };
+
+// TODO: Refactor using data transformation layer pattern when we will have API for wallets (e.g. wallet.icon)
+export const getWalletCurrencyIcon = (currency: string, is_dark_mode_on: boolean) => {
+    switch (currency) {
+        case 'demo':
+            return is_dark_mode_on ? 'IcWalletDerivDemoDark' : 'IcWalletDerivDemoLight';
+        case 'USD':
+            return 'IcCurrencyUsd';
+        case 'EUR':
+            return 'IcCurrencyEur';
+        case 'AUD':
+            return 'IcCurrencyAud';
+        case 'BTC':
+            return is_dark_mode_on ? 'IcCashierBitcoinDark' : 'IcCashierBitcoinLight';
+        case 'ETH':
+            return is_dark_mode_on ? 'IcWalletEtheriumDark' : 'IcWalletEtheriumLight';
```

Comment:
Let's be consistent here and use icons only from `wallets` folder instead of from multiple places like `wallets`, `currency` and `cashier` folders.